StripePayments/StripePayments/Source/API Bindings/Models/PaymentMethods/Types/STPPaymentMethodCashAppParams.swift

```diff
@@ -9,6 +9,7 @@
 import Foundation
 
 /// An object representing parameters used to create a CashApp Payment Method
+/// - Note: Cash App Pay is currently in beta therefore the interface of this class is subject to change.
 public class STPPaymentMethodCashAppParams: NSObject, STPFormEncodable {
```

Comment:
If it's on beta, should this be marked with `@_spi`?

Reply:
For something like bindings I don't think there is a need to mark something with SPI as control to Cash App is handled server side. E.g. no one can create an intent with Cash App if they are not a part of the beta. So we don't really realize any benefit of putting bindings behind an SPI marker. In the specific case of bindings it gets a little tricky marking things with SPI as well. 
More context here: https://stripe.slack.com/archives/CFA2HJ99A/p1673381427970729